recipes/qt/5.x.x/conanfile.py

```diff
@@ -630,6 +630,10 @@ def _getenvpath(var):
                     }) if tools.os_info.is_macos else tools.no_op():
                         self.run(self._make_program(), run_environment=True)
 
+    @property
+    def _cmake_executables_file(self):
+        return os.path.join("lib", "cmake", "Qt5Core", "executables.cmake")
```

Comment:
Sorry but I still think that this name is too generic. It might conflicts with consumers CMake files located in their own CMAKE_MODULE_PATH.